### PR TITLE
updated profile

### PR DESCRIPTION
**Pull request by benwillkommen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,12 +44,11 @@ <h1>Personal Profile</h1>
           </div>
 
           <div class="sectionContent">
-            <p>I am an experienced developer with a passion for leadership and self-improvement. I love using code to
-              solve
-              problems and empower users, learning from and mentoring teammates, and the challenge of communicating
-              complex
-              technical concepts.</p>
-            <p>On my nights and weekends, I enjoy delving into all things data science and machine learning.</p>
+            <p>I am an experienced developer with a passion for techinical leadership and software architecture.</p>
```
Comment (gilomen2) [inline index.html]: Technical is spelled wrong

File: index.html [hunk 2]
```diff
@@ -44,12 +44,11 @@ <h1>Personal Profile</h1>
...
+
+            </p>What most inspires me is solving problems at the intersection of accelerating engineers and delivering
```
Comment (gilomen2) [inline index.html:49]: Break into multiple sentences. Feels too wordy to follow.

File: index.html [hunk 3]
```diff
@@ -92,11 +91,14 @@ <h1>Work Experience</h1>
             <h2>Staff Software Engineer - Optimizely</h2>
             <p class="subDetails">August 2020 - Present</p>
             <ul class="experienceList">
-              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of an identity migration from a homegrown/Auth0 hybrid solution to Okta</li>
-              <li>Championed a solution for authentication and authorization outside of our monolith, unblocking microservice development and accelerating the entire engineering organization</li>
-              <li>Contributed to our in-house deployment platform (on Kubernetes), and drove adoption throughout the organization via consulting and pair-programming</li>
+              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of an
```
Comment (gilomen2) [inline index.html]: Remove "an" from "an identity migration"

File: index.html [hunk 4]
```diff
@@ -92,11 +91,14 @@ <h1>Work Experience</h1>
             <h2>Staff Software Engineer - Optimizely</h2>
             <p class="subDetails">August 2020 - Present</p>
             <ul class="experienceList">
-              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of an identity migration from a homegrown/Auth0 hybrid solution to Okta</li>
-              <li>Championed a solution for authentication and authorization outside of our monolith, unblocking microservice development and accelerating the entire engineering organization</li>
-              <li>Contributed to our in-house deployment platform (on Kubernetes), and drove adoption throughout the organization via consulting and pair-programming</li>
```
Comment (gilomen2) [inline index.html:97]: No parentheses around "on Kubernetes". Maybe reword to "our in-house, Kubernetes-based deployment platform"

File: index.html [hunk 5]
```diff
@@ -44,12 +44,11 @@ <h1>Personal Profile</h1>
           </div>
 
           <div class="sectionContent">
```
Comment (gilomen2) [inline index.html:46]: Nit - it doesn't really matter for published version, but css classes should be hyphenated instead of camel cased if this is just a static site

File: index.html [hunk 6]
```diff
@@ -92,11 +91,14 @@ <h1>Work Experience</h1>
...
+              <li>Championed a solution for authentication and authorization outside of our monolith, unblocking
+                microservice development and accelerating the entire engineering organization</li>
+              <li>Contributed to our in-house deployment platform (on Kubernetes), and drove adoption throughout the
+                organization via consulting and pair-programming</li>
               <li>Coached engineering teams on Domain Driven Design and facilitated Event Storming sessions</li>
-            </ul>            
+            </ul>
             <h2>Senior Software Engineer - Optimizely</h2>
             <p class="subDetails">September 2019 - August 2020</p>
             <p>I was an Optimizely customer and fan before I was an employee, and am excited to build a product
```
Comment (gilomen2) [inline index.html:102]: Update past experience to sound past tense

**Review by gilomen2 — CHANGES_REQUESTED** [review]
See comments. Content is great.

File: index.html (hunk 1 above)
Comment (benwillkommen) [inline index.html]:
```suggestion
            <p>I am an experienced developer with a passion for technical leadership and software architecture.</p>
```

File: index.html (hunk 3 above)
Comment (benwillkommen) [reply index.html]:
```suggestion
              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of
```

File: index.html [hunk 7]
```diff
@@ -92,11 +91,14 @@ <h1>Work Experience</h1>
...
             <p class="subDetails">August 2020 - Present</p>
             <ul class="experienceList">
-              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of an identity migration from a homegrown/Auth0 hybrid solution to Okta</li>
-              <li>Championed a solution for authentication and authorization outside of our monolith, unblocking microservice development and accelerating the entire engineering organization</li>
-              <li>Contributed to our in-house deployment platform (on Kubernetes), and drove adoption throughout the organization via consulting and pair-programming</li>
+              <li>Tech Lead for the Account Services Team, responsible for the technical roadmap and execution of an
+                identity migration from a homegrown/Auth0 hybrid solution to Okta</li>
+              <li>Championed a solution for authentication and authorization outside of our monolith, unblocking
+                microservice development and accelerating the entire engineering organization</li>
+              <li>Contributed to our in-house deployment platform (on Kubernetes), and drove adoption throughout the
```
Comment (benwillkommen) [inline index.html]:
```suggestion
              <li>Contributed to our in-house deployment platform on Kubernetes, and drove adoption throughout the
```
